CONTRIBUTING.md

```diff
@@ -44,24 +46,68 @@ go test -timeout=30s -p 1 ./...
 
 `run` and `run-race` targets replace `/usr/sbin/pmm-managed` and restart pmm-managed with `supervisorctl`. As a result, it will use normal filesystem locations (`/etc/victoriametrics-promscrape.yml`, `/etc/supervisord.d`, etc.) and `pmm-managed` PostgreSQL database. Other locations (inside `testdata`) and `pmm-managed-dev` database are used for unit tests.
 
+# Advanced Setup
+## Add instances for monitoring
+`make env-up` just starts the PMM server but it doesn't setup anything to be monitored. We can use [pmm-admin](https://github.com/percona/pmm-admin) and [pmm-agent](https://github.com/percona/pmm-agent) to add instances to be monitored to pmm-managed.
 
-## VSCode
-
-VSCode provides first-class support for devcontainers. See:
-
-* https://code.visualstudio.com/docs/remote/remote-overview
-* https://code.visualstudio.com/docs/remote/containers
+* Clone the pmm-admin [repo](https://github.com/percona/pmm-admin/) and install it by running `make install`.
+* Clone the pmm-agent [repo](https://github.com/percona/pmm-agent).
+* Run database instances to be monitored, you can either run your own or use the [`docker-compose.yml`](https://github.com/percona/pmm-agent/blob/master/docker-compose.yml) file provided by pmm-agent to run MySQL, PostgreSQL and MongoDB containers using `make env-up` in the pmm-agent repo (make sure to comment out the `pmm-server` service in the docker-compose file since we are already running pmm-managed in devcontainer).
+* Open another shell session and `cd` into the pmm-agent repo, run `make setup-dev` and `make run` to setup and run pmm-agent and connect it to pmm-managed
+* In another shell use pmm-admin to add agents to the database instances and start monitoring them using `pmm-admin add mysql --username=root --password=root-password`, `pmm-admin add postgresql --username=pmm-agent --password=pmm-agent-password` and `pmm-admin add mongodb --username=root --password=root-password`.
+* Now pmm-managed has started monitoring the databases, login to the web client in your browser to verify. The number of monitored instances will have increased.
 
+## Working with STT
+* Add the following environment variables to `docker-compose.yml` to dowload checks from Percona's checks service
+```
+PERCONA_TEST_SAAS_HOST=check-dev.percona.com:443
+PERCONA_TEST_CHECKS_PUBLIC_KEY=RWTg+ZmCCjt7O8eWeAmTLAqW+1ozUbpRSKSwNTmO+exlS5KEIPYWuYdX
+PERCONA_TEST_CHECKS_INTERVAL=10s # set it to any low duration to execute checks quickly
+```
+* Setup the devcontainer using `make env-up` and run your changes inside it by running `make env` and then`make run`.
+* Follow the steps in the previous section to set up instances for monitoring.
```

Comment:
```suggestion
* Follow the steps in the [Add Instances for Monitoring](#add-instances-for-monitoring) to set up instances for monitoring.
```
Will future proof in case a new section ever gets inserted between add and STT